Jenkins Maven Release Plugin throws exception

**Describe the bug**
mvn site throws an exception when building Core API. As follows:
`QuestionService.java:93: error: exception not thrown: com.gkenna.tullamoreqa.core.api.exceptions.QuestionNotFoundException`

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,6 +2,6 @@ language: java
 sudo: required
 services:
   - docker
-script: mvn checkstyle:check clean install -Ptest
+script: mvn checkstyle:check clean install javadoc:javadoc -Ptest
 
 after_success: coveralls

--- a/pom.xml
+++ b/pom.xml
@@ -3,7 +3,8 @@
   ~ Copyright (c) 2018. Gavin Kenna
   -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.gkenna.tullamoreqa</groupId>
@@ -27,6 +28,7 @@
         <url>https://github.com/GavinKenna/tullamoreqa.git</url>
         <tag>HEAD</tag>
     </scm>
+
 
     <distributionManagement>
         <repository>

--- a/tullamoreqa-core-api/src/main/java/com/gkenna/tullamoreqa/core/api/services/QuestionService.java
+++ b/tullamoreqa-core-api/src/main/java/com/gkenna/tullamoreqa/core/api/services/QuestionService.java
@@ -93,7 +93,8 @@ public interface QuestionService {
      * @throws QuestionNotFoundException Thrown if the
      *                                   {@link Question} cannot be found.
      */
-    Question getQuestion(final Long questionId);
+    Question getQuestion(final Long questionId)
+            throws QuestionNotFoundException;
 
     /**
      * Return all {@link Question}s in the DB.

--- a/tullamoreqa-core-impl/src/main/java/com/gkenna/tullamoreqa/core/impl/controllers/QuestionControllerImpl.java
+++ b/tullamoreqa-core-impl/src/main/java/com/gkenna/tullamoreqa/core/impl/controllers/QuestionControllerImpl.java
@@ -71,7 +71,15 @@ public class QuestionControllerImpl implements QuestionController {
             @PathVariable("id") final Long questionId) {
 
         LOGGER.debug("Attempting to get Question {}", questionId);
-        Question output = questionService.getQuestion(questionId);
+        Question output = null;
+        try {
+            output = questionService.getQuestion(questionId);
+        } catch (QuestionNotFoundException e) {
+            /*
+            TODO Implement correct error handling.
+             */
+            e.printStackTrace();
+        }
         if (output == null) {
             LOGGER.error("Question with id {} not found.", questionId);
             // TODO Replace this exception with custom exception

